src/main/java/juja/microservices/users/service/UserService.java

```diff
@@ -59,9 +59,15 @@ private UserDTO convertGetAllUsersDto(User user) {
         logger.debug("Received response from repository: {}", users.toString());
 
         logger.debug("Converting received user list to userDTO list");
-        List<UserDTO> result =users.stream()
-                .map(this::convertGetUuidBySlackDto)
-                .collect(Collectors.toList());
+        List<UserDTO> result;
+        try {
+            result = users.stream()
+                    .map(this::convertGetUuidBySlackDto)
+                    .collect(Collectors.toList());
+        } catch (NullPointerException e) {
```

Comment:
Honestly, I don't like it. Noone catches NullPointerException. If you want to use feature of Java8, look at the Optional class as solution. Or check `if null` with old plain method.